vipps-ecom-api.md

```diff
@@ -1810,24 +1810,43 @@ API when the payment is created ("url": "vipps://?token=eyJraWQiOiJqd3RrZXki..")
 
 Example of how to open Vipps:
 
-```java
-try {
-  PackageManager pm = context.getPackageManager();
-  PackageInfo info = pm.getPackageInfo( , PackageManager.GET_ACTIVITIES);
-  if(versionCompare(info.versionName,   ) >= 0) {
-    String uri = deeplinkURL; // Use deeplink url provided in ecom API response
-    Intent intent = new Intent(Intent.ACTION_VIEW);
-    intent.setData(Uri.parse(uri));
-    startActivityForResult(intent,requestCode);
-  } else {
-    // Notify user to download the latest version of Vipps application.
-  }
-} catch (PackageManager.NameNotFoundException e) {
-  // No Vipps app! Open play store page.
-  String url = " https://play.google.com/store/apps/details?id=no.dnb.vipps";
-  Intent storeIntent = new Intent(Intent.ACTION_VIEW);
-  storeIntent.setData(Uri.parse(url));
-  startActivity(storeIntent);
+```kotlin
+package com.example.app
+
+import android.app.Activity
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.net.Uri
+
+class MyActivity : Activity() {
+    companion object {
+        const val vippsRequestCode: Int = TODO("insert custom request code")
+        const val vippsMinVersion: String = TODO("insert min version")
```

Comment:
This seems quite handy! We should recommend that merchants specify this to make sure the user has the correct version. Maybe add a line of tips/info for that?

Reply:
It is handy, but it's something Vipps will usually handle by forcing updates when needed. Maybe it is better to leave it out to keep the example as simple as possible? Or at least comment that this is something the merchant should usually not set.

Reply:
Sure, let's remove the version check 👍